ext/dcos-installer/dcos_installer/action_lib/configure.py

```diff
@@ -27,6 +30,23 @@ def do_configure(gen_config):
     pkgpanda.write_json('/genconf/cluster_packages.json', gen_out.cluster_packages)
 
 
+def do_aws_cf_configure(gen_config):
+    subprocess.check_output(['mkdir', '-p', SERVE_DIR])
+
+    print("gen_config:")
+    pprint.pprint(gen_config)
+
+    # TODO(cmaloney): CURPOS
+    gen_out = gen.installer.aws.make_custom_aws_templates(gen_config)
+
+    # TODO(lingmann): At this point, gen_out['artifacts'] is a list of Dicts with the keys 'channel_path',
```

Comment:
Ideally we'd upload them all to s3, and there's all the code in `release` we could steal to do that from the "artifacts" form.

For now though, definitely just iterate over them, write them to <SERVE_DIR>/<channel_path>